tutordiscovery/plugin.py

```diff
@@ -1,14 +1,16 @@
 from glob import glob
 import os
+import pkg_resources
+
+from tutor import hooks as tutor_hooks
 
```

Comment:
you imported it as `tutor_hooks` here, but in [cookiecutter](https://github.com/overhangio/cookiecutter-tutor-plugin#migrating-from-v0-plugins) it used `hooks` does it matter to be that consistent?

Reply:
This is due to a weird issue in the implementation of OfficialPlugin... I commented on it during this livestream: https://youtu.be/28_lt90UREo?t=2430
tl;dw: you only have to concern yourself with this for the Nutmeg release, and only for official plugins.